trade-finance-manager-ui/component-tests/utilisation-reports/utilisation-reports.component-test.ts

```diff
@@ -58,4 +63,13 @@ describe(page, () => {
   it('should render the report reconciliation table', () => {
     getWrapper().expectElement('[data-cy="utilisation-report-reconciliation-table"]').toExist();
   });
+
+  it('should render the read only banner for PDC_READ users', () => {
+    getWrapper(true).expectElement('[data-cy="read-only-banner"]').toExist();
+    getWrapper(true)
```

Comment:
Could we use object params for setting the `isPDCReadUser` so it is clear here when calling it what this true is doing?